app/assets/stylesheets/static_pages.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```

Comment:
delete this file. because you dont use it

Reply:
Done